docs/dashboard_ingestion_guide.md

```diff
@@ -0,0 +1,119 @@
+# Dashboard Ingestion guidance 
+(Currently this guidance is about using Databuilder to ingest Dashboard metadata into Neo4j and Elasticsearch)
+
+Dashboard ingestion consists of multiple Databuilder jobs and it can be described in four steps:
+
+ 1. Ingest base data to Neo4j.
+ 2. Ingest additional data and decorate Neo4j over base data.
+ 3. Update Elasticsearch index using Neo4j data
+ 4. Remove stale data
+
+Note that Databuilder jobs need to be sequenced as 1 -> 2 -> 3 -> 4. To sequencing these jobs, Lyft uses Airflow to orchestrate the job, but Databuilder is not limited to Airflow and you can also simply use Python script to sequence it -- not recommended for production though.
+
+Also, step 1, 3, 4 is expected to have one Databuilder job where Step 2 is expected to have **multiple** Databuilder jobs and number of Databuilder jobs in step 2 is expected to grow as we add more metadata into Dashboard. To improve performance, it is recommended, but not required, to execute Databuilder jobs in step 2 concurrently.
+
+Here this documentation will be using [Mode Dashboard](https://app.mode.com/) as concrete example to show how to ingest Dashboard metadata. However, this ingestion process not limited to Mode Dashboard and any other Dashboard can follow this flow.
+
+### 1. Ingest base data to Neo4j.
```

Comment:
you mean base or basic?

Reply:
I was trying to use `base` as it's a `base` data that needs to be ingested prior to other data. WDYT?